### DISCUSSION TITLE
Bellows CONFIG_TX_POWER_MODE ConfigurationValue

### DISCUSSION THREAD

**Discussion by pipiche38** [zigpy/zigpy#1202] [Q&A] [answered]
I'm struggling to understand how I should set the CONFIG_TX_POWER_MODE configuration value when using bellows.

I feel that it has changed within the previous release where Config parameters went from low layers to zigpy, but I didn't see any equivalent to that one .

Under zigpy_znp there is a function set_tx_power() which is used to set a specific power, but what about bellows ?

**Comment by puddly** [comment]
You would use the `setRadioPower` command:

> Name: setRadioPower ID: 0x0099
> ---
> Description: Sets the radio output power at which a node is operating. Ember radios have discrete power settings. For a list of available power settings, see the technical specification for the RF communication module in your Developer Kit. **Note: Care should be taken when using this API on a running network, as it will directly impact the established link qualities neighboring nodes have with the node on which it is called. This can lead to disruption of existing routes and erratic network behavior.**

Feel free to make a PR with this change to mimic the ZNP API.

**Reply by pipiche38** [reply]
I'm not sure to follow you, do you mean that the functionality has been removed from a previous version of bellows (as it works before ) ?

**Reply by pipiche38** [reply, marked as answer]
I'm pushing the corresponding EZSP Config Parameter when starting the Coordinator
